vllm/worker/cpu_worker.py

```diff
@@ -138,6 +138,10 @@ def __init__(
         self.is_driver_worker = is_driver_worker
         if self.is_driver_worker:
             assert self.rank == 0, "The driver worker must have rank 0."
+        init_cached_hf_modules = self.model_config.trust_remote_code
+        if init_cached_hf_modules:
+            from transformers.dynamic_module_utils import init_hf_modules
+            init_hf_modules()
```

Comment:
nit: good to put this new common logic into a util method, with docstring

Reply:
fixed in [2509db4](https://github.com/vllm-project/vllm/pull/4024/commits/2509db4e62c44192198de1b61c9eda8dd75628a6)